it/controllers/ReportDeadlinesControllerISpec.scala

```diff
@@ -17,150 +17,372 @@ package controllers
 
 import java.time.LocalDate
 
-import helpers.{ComponentSpecBase, GenericStubMethods}
+import config.FrontendAppConfig
 import helpers.IntegrationTestConstants.GetReportDeadlinesData._
 import helpers.IntegrationTestConstants._
-import helpers.servicemocks.{AuthStub, BtaPartialStub, SelfAssessmentStub, UserDetailsStub}
-import org.scalatest.Assertion
+import helpers.servicemocks.SelfAssessmentStub
+import helpers.{ComponentSpecBase, GenericStubMethods}
 import play.api.http.Status._
-import play.api.libs.json.{JsNull, JsValue}
-import play.api.libs.ws.WSResponse
 import utils.ImplicitDateFormatter
 
 class ReportDeadlinesControllerISpec extends ComponentSpecBase with ImplicitDateFormatter with GenericStubMethods {
 
+  lazy val appConfig: FrontendAppConfig = app.injector.instanceOf[FrontendAppConfig]
+
   "Calling the ReportDeadlinesController" when {
 
-    "isAuthorisedUser with an active enrolment" which {
+    "the ReportDeadlines Feature is disabled" should {
 
-      "has a single business obligation" should {
+      "Redirect to the Income Tax View Change Home Page" in {
 
-        "display a single obligation with the correct dates and status" in {
+        appConfig.features.reportDeadlinesEnabled(false)
 
-          isAuthorisedUser(true)
-          stubUserDetails()
-          getBizDeets(GetBusinessDetails.successResponse(testSelfEmploymentId))
-          getPropDeets(GetPropertyDetails.successResponse())
+        isAuthorisedUser(true)
+        stubUserDetails()
+        getBizDeets(GetBusinessDetails.successResponse(testSelfEmploymentId))
+        getPropDeets(GetPropertyDetails.successResponse())
 
-          And("I wiremock stub a single business obligation response")
-          SelfAssessmentStub.stubGetBusinessReportDeadlines(testNino, testSelfEmploymentId, singleReportDeadlinesDataSuccessModel)
+        And("I wiremock stub a single business obligation response")
+        SelfAssessmentStub.stubGetBusinessReportDeadlines(testNino, testSelfEmploymentId, singleReportDeadlinesDataSuccessModel)
 
-          When("I call GET /report-quarterly/income-and-expenses/view/obligations")
-          val res = IncomeTaxViewChangeFrontend.getReportDeadlines
+        When("I call GET /report-quarterly/income-and-expenses/view/obligations")
+        val res = IncomeTaxViewChangeFrontend.getReportDeadlines
 
-          verifyBizDeetsCall()
-          verifyPropDeetsCall()
-          verifyBizObsCall(testSelfEmploymentId)
+        Then("the result should have a HTTP status of SEE_OTHER (303) and redirect to the Income Tax home page")
+        res should have(
+          httpStatus(SEE_OTHER),
+          redirectURI("/report-quarterly/income-and-expenses/view")
```

Comment:
Not sure it matters, but would `redirectURI(controllers.routes.HomeController.home().url)` be better?